analyzer/consensus/genesis.go

```diff
@@ -1,6 +1,6 @@
-// Package generator generates migrations for the Oasis Indexer
-// from the genesis file at a particular height.
-package generator
+// Conversion of the genesis state into SQL statements.
+
+package consensus
```

Comment:
🤔 but this file is `genesis.go` not say `migrator.go`?

Reply:
The package name comes from the folder name. The filename comes from the fact that within the `analyzer/consensus` package, this file provides analysis of the genesis part of the consensus layer.

Reply:
I thought you went for a more exact refactor/renaming from the `generator.go`. Cool.